docs/07_code_coverage_artifact_import.md

```diff
@@ -0,0 +1,347 @@
+# CI artifacts import
+> Summary of the proposed change
+
+Fetch code coverage data using the CI artifact API.
+
+# References
+> Link to supporting documentation, GitHub tickets, etc.
+
+- [Bitrise](https://devcenter.bitrise.io/).
+- [Buildkite](https://buildkite.com/docs/tutorials/getting-started).
+- [CircleCI](https://circleci.com/docs/).
+- [Jenkins](https://jenkins.io/doc/).
+
+# Motivation
+> What problem is this project solving?
+
+Automatically collect code coverage metrics from various CI providers and build jobs.
+
+# Goals
+> Identify success metrics and measurable goals.
+
+* CI can export coverage metric as an artifact.
+* Exported artifacts are accessible via the CI's API.
+
+# Non-Goals
+> Identify what's not in scope.
+
+Implementing code coverage artifact creation is out of scope because that's job specific.
```

Comment:
I think this means that the process is affected by all of these points (I'd even say, by a programming language as well).
How about this: 
`because it depends on CI, job, code coverage tool and even programming language.`

However, I don't think we should explain what the `job` means here since the process varies from case to case.

Reply:
Thanks, I think your suggestion should work.